frontend/src/lib/components/media_card.svelte

```diff
@@ -56,6 +56,11 @@
       </h2>
     </figure>
   {/if}
+  {#if media.imdb_rating}
+    <div class="absolute top-0 left-0 mx-1 -mt-1 opacity-85">
```

Comment:
Not sure if it does anything
```suggestion
    <div class="absolute top-0 left-0 mx-1 mt-1 opacity-85">
```

Reply:
Ahh, okay - Didn't know you could do that